README.md

```diff
@@ -16,6 +22,7 @@ sensor:
   - platform: sems
     username: 'XXXX'
     password: 'XXXX'
+    id : '12345678-1234-1234-1234-123456789012'
```

Comment:
I would prefer use of station_id instead of ID. It is more self describing.

CONF_STATION_ID

Reply:
Hi!

I agree, makes more sense. I modified it in this PR.